### ISSUE TITLE
Audio issue after  reversed the Question and Answer 

### ISSUE THREAD

**Issue by sepeagle** [helloworld1/AnyMemo#374]
I found this audio problem as following:

1. I reversed the Q&A  by customizing the card field 1 and 2, for instance, I choose Answer for card field 1, and Question for card field 2.
2. When I study the flashcard,  I expect to firstly hear the Audio for Answer, and secondly for Questions. However, in fact it comes up firstly with Audio for original Questions, and secondly for original Answer. 
In a word, I reversed the Q&A, but the audio are not following.

I don't want to export the Q&A list and reverse it by EXCEL.  Is there any solution of Anymemo itself to fix this fault ?

Thanks in advance for any idea.

**Comment by helloworld1** [comment]
Yes, that is a bug. The audio should go with fields not question / answer. However, you can swap the question / answer without using excel. Before everything back up you db file in case something goes wrong.
First, go to "Preview / edit". Click "..." to invoke the menu, click "Advanced..." and select "Swap the Q/A of all cards" and select "SWAP". The question and answer will be reversed and the audio will be correct too.

**Comment by sepeagle** [comment]
It works ! Thanks !

**Comment by helloworld1** [comment]
Glad to hear. I will keep this bug open to track the bug you mentioned. The audio should be linked with field not q/a.
